disttask/framework/dispatcher/dispatcher.go

```diff
@@ -171,6 +171,7 @@ func (d *dispatcher) DispatchTaskLoop() {
 				if d.isRunningGTask(gTask.ID) {
 					continue
 				}
+				// owner changed
```

Comment:
What does it mean?

Reply:
tidb owner changed, so just `setRunningGTask`